### PR TITLE
modified-adaptive

### PR DESCRIPTION
**Pull request by NataDav-create**

### CODE REVIEW

File: src/components/DownloadButton/DownloadButton.styles.js [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
-  letter-spacing: 0.36px;
-  cursor: pointer;
-  background: ${styleBg === 'primary'
-    ? 'linear-gradient(16deg, #bb9b66 0%, #c6aa71 48%, #e9d793 86%, #d9c184 100%)'
-    : '#121212'};
-  border: none;
-  border-radius: 40px;
-  outline: none;
-  transform: translateY(50%);
+  button {
```
Comment (a-guris) [inline src/components/DownloadButton/DownloadButton.styles.js]: let's avoid using tag selectors and use classes instead

File: src/components/Layout/Layout.styles.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { css } from '@emotion/react';
 
 const styles = css`
   .container {
+    position: relative;
     div {
```
Comment (a-guris) [inline src/components/Layout/Layout.styles.js:7]: code style: add empty line before ruleset

File: src/styles/colors.js [hunk 3]
```diff
@@ -4,4 +4,5 @@ export default {
   bgWhite: '#F9F8F5',
   gold: '#A39161',
   btnGrey: '#121212',
+  menuColor: '#0d0d0d',
```
Comment (a-guris) [inline src/styles/colors.js]: capital letter

File: src/styles/index.js [hunk 4]
```diff
@@ -42,3 +42,6 @@ export const aboveTablet = getAboveBP(breakpoints.tablet);
 
 export const belowMedium = getBelowBP(breakpoints.medium);
 export const aboveMedium = getAboveBP(breakpoints.medium);
+
```
Comment (a-guris) [inline src/styles/index.js]: only one empty line at end of file

**Review by a-guris — CHANGES_REQUESTED** [review]

**Comment by NataDav-create** [comment]
working over some mistakes

**Pull request closed without merging**